packages/devtools_app/lib/src/inspector/flutter/inspector_tree_flutter.dart

```diff
@@ -83,18 +84,21 @@ class _InspectorTreeRowState extends State<_InspectorTreeRowWidget>
 
 class InspectorTreeControllerFlutter extends Object
     with InspectorTreeController, InspectorTreeFixedRowHeightController {
+  // Controller for controlling custom animation in the Tree
+  AnimationController animationController;
```

Comment:
@DaveShuckerow should we try to keep AnimationController classes out of our controllers?